core/templates/pages/creator-dashboard-page/modal-templates/upload-activity-modal.controller.ts

```diff
@@ -36,10 +43,5 @@ angular.module('oppia').controller('UploadActivityModalController', [
 
       $uibModalInstance.close(returnObj);
     };
-
-    $scope.cancel = function() {
-      $uibModalInstance.dismiss('cancel');
-      AlertsService.clearWarnings();
```

Comment:
Hmmm, why remove this?

Reply:
Because I'm handling the cancel action with generic modal, so what I did is to move the `clearWarnings` method to the promise rejection callback in `$uibModal.open`method which is called after `$uibModalInstance.dismiss` executes.

Reply:
Could you please test this locally..

Reply:
Yes, I've tested it, it's working!